CMakeLists.txt

```diff
@@ -24,67 +26,10 @@ find_package(Jasper REQUIRED)
 find_package(PNG REQUIRED)
 find_package(ZLIB REQUIRED)
 
```

Comment:
a `find_package(Jasper)` is missing here.

Reply:
It's there, right above the PNG line